src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -204,11 +213,14 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
     if (isFirstE2EIActivation || firingDate <= Date.now()) {
       // We want to automatically trigger the enrollment modal if it's a devices in team that just activated e2eidentity
       // Or if the timer is supposed to fire now
-      void task();
+      void task(isSnoozable);
     } else {
       LowPrecisionTaskScheduler.addTask({
         key: timerKey,
-        task,
+        task: () => {
+          const {isSnoozable} = getEnrollmentTimer(identity, e2eActivatedAt, this.config.gracePeriodInMs);
+          return task(isSnoozable);
+        },
```

Comment:
We recalculate the isSnoozable value, to prevent two modals being displayed one after the other in case the cert has expired. 

Previously:
1. snoozable: your cert will expire soon -> remind me later
2. (immediately) not snoozable: your cert has expired

Now:
1. (immediately) not snoozable: your cert has expired